odbc/src/api/functions.rs

```diff
@@ -256,8 +300,8 @@ pub unsafe extern "C" fn SQLBulkOperations(
 /// Because this is a C-infereface, this is necessarily unsafe
 ///
 #[no_mangle]
-pub unsafe extern "C" fn SQLCancel(_statement_handle: HStmt) -> SqlReturn {
-    unimplemented!()
+pub unsafe extern "C" fn SQLCancel(statement_handle: HStmt) -> SqlReturn {
+    panic_safe_exec!(|| { unimplemented!() }, statement_handle);
```

Comment:
[nit] We could define another macro call `unimpl` (or similar) that does this line.